Description: Fix build against external libtalloc
 Upstream Makefile is untested with builds against external libtalloc; fix a
 make error when libtalloc is not being built.
Author: Steve Langasek <vorlon@debian.org>

Index: b/source3/configure.in
===================================================================
--- a/source3/configure.in	2009-11-08 15:54:25.000000000 +0100
+++ b/source3/configure.in	2009-11-08 15:55:38.000000000 +0100
@@ -42,6 +42,8 @@
 if test "x$enable_external_libtalloc" != xyes
 then
 	m4_include(../lib/talloc/libtalloc.m4)
+else
+	tallocdir=../lib/talloc
 fi
 
 m4_include(../lib/tevent/libtevent.m4)
